settings/staging.py

```diff
@@ -1,13 +1,11 @@
 from .prod import *  # noqa: ignore=F405
 
-ALLOWED_HOSTS = [
-    "staging.eval.ai",
-]
+ALLOWED_HOSTS = ["staging.eval.ai"]
 
 CORS_ORIGIN_ALLOW_ALL = False
 
 CORS_ORIGIN_WHITELIST = (
     "staging-evalai.s3.amazonaws.com",
     "evalai-staging.cloudcv.org",
```

Comment:
@RishabhJain2018  we don't have to whitelist `staging.eval.ai`?

Reply:
good catch, will update this. Thanks!